src/modules/_nomodule/PixelManipulation.js

```diff
@@ -32,7 +32,13 @@ module.exports = function PixelManipulation(image, options) {
     // https://github.com/p-v-o-s/infragram-js/blob/master/public/infragram.js#L173-L181
 
 
-
+    if (!options.inBrowser && !process.env.TEST && options.ui) {
+      try {
+        var pace = require('pace')(pixels.shape[0] * pixels.shape[1]);
+      } catch (e) {
+        options.inBrowser = true;
```

Comment:
Is this a good assumption? If our error is `TypeError: fs.readFileSync is not a function` shouldn't we try to work around it by disabling `ui` as @tech4GT suggested?

Reply:
Disbling ui is not actually working here.